skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
@soupault this function does something useful when applying two different thresholds, perhaps obtained through different means (e.g. `threshold_otsu` with different offsets, or `threshold_sauvola` with different k). It is a *downstream* function from `threshold_*`. There is no such function as `threshold_hysteresis` because it is not a function that returns a threshold value/mask, but rather a function that applies a predetermined pair values/masks.